lib/consumerGroupStream.js

```diff
@@ -33,11 +33,39 @@ class ConsumerGroupStream extends Readable {
     super({ objectMode: true, highWaterMark: options.highWaterMark || DEFAULT_HIGH_WATER_MARK });
 
     _.defaultsDeep(options || {}, DEFAULTS);
+    const self = this;
 
     this.autoCommit = options.autoCommit;
 
     options.connectOnReady = false;
     options.autoCommit = false;
+    const originalOnRebalance = options.onRebalance;
+    options.onRebalance = function (isAlreadyMember, callback) {
+      let autoCommitCalled = false;
+      const autoCommit = function (err) {
```

Comment:
could warp this in `_.once` to avoid having a flag.

Reply:
done